mains/parse_metadata.py

```diff
@@ -98,3 +100,34 @@
     X_tmp, Y_tmp, train_size=size_val, random_state=1
 )
 
+#load images
+def load_image(addr):
+    img = cv.imread(addr)
+    #img = cv.resize(img, (224, 224), interpolation=cv.INTER_CUBIC)
+    img = cv.cvtColor(img, cv.COLOR_BGR2RGB)
+    img = img.astype(np.float32)
+    return img
+
+
+#convert data to features
+def _int64_feature(value):
```

Comment:
maybe you should choose a different function name here.
https://stackoverflow.com/questions/1301346/what-is-the-meaning-of-a-single-and-a-double-underscore-before-an-object-name

Reply:
Just copy-pasted from a tutorial, again: only used once no need to apply OO-Patterns IMHO.